tests/unit/test_collector.py

```diff
@@ -371,16 +391,15 @@ def test_group_locations__non_existing_path():
     assert not urls and not files, "nothing should have been found"
 
 
-def make_fake_html_page(url):
+def make_fake_html_response(url):
     html = dedent(u"""\
     <html><head><meta name="api-version" value="2" /></head>
     <body>
     <a href="/abc-1.0.tar.gz#md5=000000000">abc-1.0.tar.gz</a>
     </body></html>
     """)
     content = html.encode('utf-8')
-    headers = {}
-    return HTMLPage(content, url=url, headers=headers)
+    return pretend.stub(content=content, url=url, headers={})
```

Comment:
No action needed, just noting that this is a `requests.Response` stub. I think it's clear enough as-is from its use in patching `_get_html_response` and probably doesn't need a comment or anything - it will probably only be confusing for someone looking at this PR that isn't already familiar with the `collector` module.

Reply:
It doesn't hurt to add a comment, so I'll do that. Thanks for the suggestion.